### PR TITLE
pngcheck: add page

### PR DESCRIPTION
**Pull request by ajrequenez**
- [x] The page(s) are in the correct platform directories: `linux`
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**3.0.3

Although not listed in the pen-testing tools, this is a useful forensics utility used in CTFs.

Related to #6863

### CODE REVIEW

File: pages/linux/pngcheck.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# pngcheck
+
+> Useful forensics tool for validating the integrity of image files.
+> Can also extract embedded images and text from the file.
+> More information: <http://www.libpng.org/pub/png/apps/pngcheck.html>.
+
+- Verify the integrity of an image file:
+
+`pngcheck {{image}}`
```
Comment (navarroaxel) [inline pages/linux/pngcheck.md:9]: Are we talking about images like the created with the `dd` command? Perhaps this better
```suggestion
`pngcheck {{file.img}}`
```

Comment (ajrequenez) [reply pages/linux/pngcheck.md]: Thank you for the feedback.  I hadn't thought about the ambiguity of the term.  It's actually for PNG image files.  I have updated the description to include (.png | .jng | .mng) image types and changed file.png in the examples.

File: pages/linux/pngcheck.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# pngcheck
+
+> Useful forensics tool for validating the integrity of PNG based (.png|.jng|.mng) image files.
+> Can also extract embedded images and text from the file.
```
Comment (navarroaxel) [inline pages/linux/pngcheck.md:4]:
```suggestion
> Can also extract embedded images and text from a file.
```

File: pages/linux/pngcheck.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- Verify the integrity of an image file:
+
+`pngcheck {{file.png}}`
+
+- Check the file with [v]erbose and [c]olorized output:
+
+`pngcheck -vf {{file.png}}`
+
+- Display contents of [t]ext chunks and [s]earch for PNGs within the file:
```
Comment (navarroaxel) [inline pages/linux/pngcheck.md:15]:
```suggestion
- Display contents of [t]ext chunks and [s]earch for PNGs within a specific file:
```

File: pages/linux/pngcheck.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+`pngcheck -ts {{file.png}}`
+
+- Search for, and e[x]tract embedded PNGs within the file:
```
Comment (navarroaxel) [inline pages/linux/pngcheck.md:19]:
```suggestion
- Search for, and e[x]tract embedded PNGs within a specific file:
```

File: pages/linux/pngcheck.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# pngcheck
+
+> Useful forensics tool for validating the integrity of PNG based (.png|.jng|.mng) image files.
+> Can also extract embedded images and text from the file.
+> More information: <http://www.libpng.org/pub/png/apps/pngcheck.html>.
+
+- Verify the integrity of an image file:
+
+`pngcheck {{file.png}}`
```
Comment (navarroaxel) [inline pages/linux/pngcheck.md:9]: We use `path/to/file` as placeholder
```suggestion
`pngcheck {{path/to/file.png}}`
```

File: pages/linux/pngcheck.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+- Check the file with [v]erbose and [c]olorized output:
+
+`pngcheck -vf {{file.png}}`
```
Comment (navarroaxel) [inline pages/linux/pngcheck.md:13]:
```suggestion
`pngcheck -vf {{path/to/file.png}}`
```

File: pages/linux/pngcheck.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+- Display contents of [t]ext chunks and [s]earch for PNGs within the file:
+
+`pngcheck -ts {{file.png}}`
+
+- Search for, and e[x]tract embedded PNGs within the file:
+
+`pngcheck -x {{file.png}}`
```
Comment (navarroaxel) [inline pages/linux/pngcheck.md:21]:
```suggestion
`pngcheck -x {{path/to/file.png}}`
```

File: pages/linux/pngcheck.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+# pngcheck
+
+> Useful forensics tool for validating the integrity of PNG based (.png|.jng|.mng) image files.
```
Comment (navarroaxel) [inline pages/linux/pngcheck.md:3]:
```suggestion
> Forensics tool for validating the integrity of PNG based (`.png`, `.jng`, `.mng`) image files.
```

**Review by navarroaxel — APPROVED** [review]

File: pages/linux/pngcheck.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+> Can also extract embedded images and text from a file.
+> More information: <http://www.libpng.org/pub/png/apps/pngcheck.html>.
+
+- Verify the integrity of an image file:
+
+`pngcheck {{path/to/file.png}}`
+
+- Check the file with [v]erbose and [c]olorized output:
+
+`pngcheck -vf {{path/to/file.png}}`
```
Comment (pixelcmtd) [inline pages/linux/pngcheck.md:13]: ![\[c\]](https://user-images.githubusercontent.com/35269695/196065072-4b89b0a8-7633-4604-8b7c-87a09d1378f9.png)
-vf

Am I missing something?

**Comment by pixelcmtd** [comment]
@ajrequenez Your first commit still has the email `jrequenez@Jamess-MacBook-Pro-3.local` in it, which CLA bot can't check. Can you please use some `git` magic (I think you need to `squash`) to fix that?

File: pages/linux/pngcheck.md (hunk 9 above)
Comment (ajrequenez) [reply pages/linux/pngcheck.md]: Fixed in commit 
e9e28b0

Thank you

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**